tests/conftest.py

```diff
@@ -179,3 +179,13 @@ def remove_sub_files_folders(path):
         for root, dirs, files in os.walk(path):
             for file in files:
                 os.remove(os.path.join(root, file))
+
+
+def get_cats(processor, col):
```

Comment:
Nice cleanup.

Are any tests actually useing the `else` block here anymore?

Reply:
I've changed a couple of stat ops to test both the single/multi gpu versions temporarily (to get code coverage requirements higher) - but will remove this soon